bats/fb-test-puppet.bats

```diff
@@ -79,12 +79,12 @@ fi
 }
 
 @test "assign puppet class to host" {
-  id=$(hammer --csv puppet-class list --search 'name = ntp' | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv puppet-class list --search 'name = chrony' | tail -n1 | cut -d, -f1)
   pc_ids=$(hammer host update --help | awk '/class-ids/ {print $1}')
   hammer host update $pc_ids $id --name $(hostname -f)
 }
 
 @test "apply class with puppet agent" {
   puppet agent -v -o --no-daemonize
-  grep -i puppet /etc/ntp.conf
+  grep -i puppet /etc/chrony.conf
```

Comment:
Turns out this module doesn't have the word `puppet` in the resulting file causing the tests to fail. To be clear, the test was to check for https://github.com/puppetlabs/puppetlabs-ntp/blob/29929d1f425e40134c51b9d11a87c0f71fcffab6/templates/ntp.conf.epp#L1 in the resulting file.